pkg/server/config.go

```diff
@@ -47,6 +47,8 @@ type HandlerConfig struct {
 type MappingHandlerConfig struct {
 	// DefaultResourceScope is the default resource scope to search resources.
 	// This is only used for global resources such as GCS bucket.
+	// Please make sure your run-pmap service account was granted the permission
```

Comment:
Nits: Please make sure the Service Account used in the Cloud Run service for Data Mapping is granted the `roles/cloudasset.viewer` to the corresponding scope level.

 run-pmap service account is confusing for people who are not familiar with the implementations details of  our terraform module

Reply:
Thanks! Updated.